igor-web/src/test/groovy/com/netflix/spinnaker/igor/docker/DockerMonitorSpec.groovy

```diff
@@ -125,6 +126,8 @@ class DockerMonitorSpec extends Specification {
           assert artifacts.size() == 1
           assert artifacts[0].name == "repository"
           assert artifacts[0].type == "DOCKER"
+          assert artifacts[0].metadata.buildNumber == "111"
+          assert artifacts[0].metadata.commitId == "ab12c3"
```

Comment:
it might be good to have a test to make sure that the keys aren't present if the newt metadata doesn't exist